static/scripts/geometry.mjs

```diff
@@ -0,0 +1,24 @@
+import { imageDimensions, imageTags } from '../data/index.mjs'
+
+const numOfTags = Object.keys(imageTags).length;
+
+const getDistance = (horizontal,vertical) => ((horizontal**2+vertical**2)**(1/2));
+
+const calcArcsin = (a1,a2) => Math.asin(Math.abs(a1)/getDistance(a1,a2));
+
+const getAngle = (horizontal,vertical) => {
+  if(horizontal>=0 && vertical>=0) return calcArcsin(horizontal,vertical)
```

Comment:
can we evaluate inside these if statement functions to abstract the reader away from the actually comparisons? Faster to read and more maintainable

Reply:
agree, it´s already been refactored.